client/trino-client/src/main/java/io/trino/client/OkHttpUtil.java

```diff
@@ -206,7 +208,10 @@ public static void setupSsl(
 
             // load TrustStore if configured, otherwise use KeyStore
             KeyStore trustStore = keyStore;
-            if (trustStorePath.isPresent()) {
+            if (useSystemTrustStore) {
+                trustStore = loadSystemTrustStore(trustStoreType);
+            }
+            else if (trustStorePath.isPresent()) {
```

Comment:
Is it possible for someone to set both the trustStorePath and useSystemTrustStore? Is there validation that will prevent this from happening, or any documentation that explains how this is handled?

Reply:
Yes, it's possible and it really should not be since it doesn't make much sense, I'll add a validation check to avoid this scenario